src/status_im/switcher/bottom_tabs.cljs

```diff
@@ -35,33 +36,18 @@
       :wallet-stack      (reset! load-wallet-tab? true)
       :browser-stack     (reset! load-browser-tab? true))))
 
-;; TODO(parvesh) - reimplement tab with counter, once design is complete
-(defn bottom-tab [icon stack-id icons-only? shared-values]
-  [:f>
-   (fn []
-     (let [bottom-tab-original-style {:padding 16}]
-       (if icons-only?
-         [rn/touchable-opacity {:active-opacity 1
-                                :style          bottom-tab-original-style
-                                :on-press       #(bottom-tab-on-press shared-values stack-id)}
-          [reanimated/view {:style (reanimated/apply-animations-to-style
-                                    {:opacity (get
-                                               shared-values
-                                               (get constants/tabs-opacity-keywords stack-id))}
-                                    {})}
-           [icons/icon icon (styles/bottom-tab-icon :bottom-tabs-selected-tab)]]]
-         [rn/view {:style bottom-tab-original-style}
-          [icons/icon icon (styles/bottom-tab-icon :bottom-tabs-non-selected-tab)]])))])
-
-(defn tabs [shared-values icons-only?]
-  [rn/view {:style (styles/bottom-tabs icons-only?)}
-   [bottom-tab :main-icons/redesign-communities24 :communities-stack icons-only? shared-values]
-   [bottom-tab :main-icons/redesign-messages24 :chats-stack icons-only? shared-values]
-   [rn/view {:width 50}]
-   [bottom-tab :main-icons/redesign-wallet24 :wallet-stack icons-only? shared-values]
-   [bottom-tab :main-icons/redesign-browser24 :browser-stack icons-only? shared-values]])
+(defn bottom-tab [icon stack-id shared-values]
+  [bottom-nav-tab/bottom-nav-tab
+   {:icon             icon
+    :icon-color-anim  (get
+                       shared-values
+                       (get constants/tabs-opacity-keywords stack-id))
+    :on-press         #(bottom-tab-on-press shared-values stack-id)}])
 
 (defn bottom-tabs [shared-values]
-  [:<>
-   [tabs shared-values false]
-   [tabs shared-values true]])
+  [rn/view {:style (styles/bottom-tabs-container false)}
```

Comment:
why extra view ?

Reply:
![image](https://user-images.githubusercontent.com/17097240/193805052-299cc5b3-d636-4f0d-b0d6-4a93a5c96534.png)

First view is only for bottom tabs, which will have fix size and position. Other view is extended bottom tab.
(I can look into this more, but view hierarchy is currently more like figma)